src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js

```diff
@@ -55,14 +55,15 @@ const ReportsTrafficSources = function({ domainName, dateFrom, dependencies: { d
                     />
                   </h6>
                   <span>
-                    {trafficSource.quantity} (
+                    {trafficSource.quantity}
                     <span>
+                      (
```

Comment:
Just curiosity, why did you move the parenthesis inside the span?

Reply:
the design required a space between `span` and `span`. By having the `(` outside the span it was separated into three parts, and it looked bad